CONTRIBUTING.md

```diff
@@ -17,4 +17,12 @@ Atlassian requires contributors to sign a Contributor License Agreement, known a
 Prior to accepting your contributions we ask that you please follow the appropriate link below to digitally sign the CLA. The Corporate CLA is for those who are contributing as a member of an organization and the individual CLA is for those contributing as an individual.
 
 * [CLA for corporate contributors](https://opensource.atlassian.com/corporate)
-* [CLA for individuals](https://opensource.atlassian.com/individual)
\ No newline at end of file
+* [CLA for individuals](https://opensource.atlassian.com/individual)
+
+## Code Owners' Responsibilities
```

Comment:
We could explicitly state that we don't accept any changes to the Github Actions workflow files.